### PR TITLE
Create dronebl.org.xml

### PR DESCRIPTION
**Pull request by Tea23**

### CODE REVIEW

File: src/chrome/content/rules/dronebl.org.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="DroneBL">
+        <target host="dronebl.org" />
+        <target host="www.dronebl.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/dronebl.org.xml]: Please also add:
```
eu1.dronebl.org
frontend1.dronebl.org
mirror1.dronebl.org
```
and a comment about the non-working host `lists.dronebl.org`.

Our code style uses tabs to indent lines.

**Comment by Bisaloo** [comment]
@Tea23 are you still interested in working on this?

File: src/chrome/content/rules/dronebl.org.xml (hunk 1 above)
Comment (pipboy96) [reply src/chrome/content/rules/dronebl.org.xml]: `lists.dronebl.org` -> NXDOMAIN.

Comment (pipboy96) [reply src/chrome/content/rules/dronebl.org.xml]: `frontend1.dronebl.org` is also NXDOMAIN.

**Comment by pipboy96** [comment]
All targets are preloaded.

**Pull request closed without merging**